File: packages/dd-trace/test/llmobs/parity/scenarios/openai-agents/tool-calls.py
import asyncio

from agents import Runner

from _agents_common import finish
from _agents_common import addition_agent
from _agents_common import setup


async def main():
    setup()
    await Runner.run(addition_agent(), "What is the sum of 1 and 2?")
    finish()


asyncio.run(main())

File: packages/dd-trace/test/llmobs/parity/scenarios/openai-agents/tool-calls.js
'use strict'

const { additionAgent, setup } = require('./common')

async function main () {
  const { agents, finish } = setup()
  await agents.run(additionAgent(), 'What is the sum of 1 and 2?')
  await finish()
}

main().catch(error => {
  process.stderr.write(`${error.stack ?? error}\n`)
  process.exitCode = 1
})
